bn254/src/pairing.rs

```diff
@@ -106,15 +117,11 @@ pub fn alt_bn128_versioned_pairing(
 pub fn alt_bn128_pairing(input: &[u8]) -> Result<Vec<u8>, AltBn128Error> {
     #[cfg(not(target_os = "solana"))]
     {
-        alt_bn128_versioned_pairing(VersionedPairing::V0, input, Endianness::BE)
+        alt_bn128_versioned_pairing(VersionedPairing::V1, input, Endianness::BE)
     }
     #[cfg(target_os = "solana")]
     {
-        if input
-            .len()
-            .checked_rem(ALT_BN128_PAIRING_ELEMENT_SIZE)
-            .is_none()
-        {
+        if input.len() % ALT_BN128_PAIRING_ELEMENT_SIZE != 0 {
```

Comment:
Let's change this to the clippy style as was done in `alt_bn128_versioned_pairing`.

Reply:
Oh I see. Sorry, I was also wondering why you reverted there. I think the changes are good as is then.